server/routes/matchChat.ts

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express';
+import {
+  addMessage,
+  deleteMessage,
+  editMessage,
+  getAllMessages,
+  getMessageById,
+} from '../controllers';
+
+const chatRouter: Router = Router();
+
+chatRouter.get('/', getAllMessages);
+chatRouter.get('/:id', getMessageById);
+chatRouter.post('/add', addMessage);
```

Comment:
`chatRouter.post('/add', addMessage);`
`add` not necessary 
for best practice to in router naming convention 
`chatRouter.post('/', addMessage);`